lib/ui/radio/radio_screen.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:mcnd_mobile/di/providers.dart';
+import 'package:mcnd_mobile/ui/shared/hooks/use_once.dart';
+
+class RadioScreen extends HookWidget {
+  const RadioScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final viewModel = useProvider(radioViewModelProvider);
+    final state = useProvider(radioViewModelProvider.state);
+    useOnce(() => viewModel.load());
+    return Scaffold(
+      appBar: AppBar(
+        title: const AutoSizeText(
+          'Quran Radio',
+          maxLines: 1,
+          maxFontSize: 30,
+          minFontSize: 15,
+        ),
+      ),
+      body: SizedBox.expand(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            const Text(
+              'Egypt Quran Karim Station',
```

Comment:
Just use "Quran Kareem Radio"

Reply:
and please use Autoscaling text

Reply:
I think it's not needed here
![Screen Shot 2021-04-24 at 2 38 46 AM](https://user-images.githubusercontent.com/6633545/115941578-33789900-a4a6-11eb-8f9f-2da3a359c20f.png)

Reply:
I think it's not needed here (This is on iPhone SE)
![Screen Shot 2021-04-24 at 2 38 46 AM](https://user-images.githubusercontent.com/6633545/115941578-33789900-a4a6-11eb-8f9f-2da3a359c20f.png)